src/LSP/language-features/completions.jl

```diff
@@ -499,3 +501,159 @@ presented in the editor.
     """
     items::Vector{CompletionItem}
 end
+
+@interface CompletionClientCapabilities begin
```

Comment:
Right now, we don't change how the LS behaves based on client capabilities, so maybe we don't need to add this yet. I think we'll need it later, for example, when we implement lazy completion. If a frontend doesn't support it, we might need to fall back to eager completion. Anyway, it's fine to add it. I'll just make a note of it for when we add new features later.